theme-shop/README.md

```diff
@@ -0,0 +1,14 @@
+# Nintendo 3DS Theme Shop
+## Prerequisites
+* Python
+* libcurl 7.19.0 or greater
+* 3DS Common Prod Cert from any 3DS system
+* boot9.bin from any 3DS system
+## Usage
+1. Set these environment variables, like so:
+```bash
+set CTR_PROD_3=/path/to/your/3ds_common_cert.pem
+set BOOT9_PATH=/path/to/your/boot9.bin
+```
+2. Run `pip install -r requirements.txt` to install dependencies
+3. Run `python3 download.py`
```

Comment:
Could there be some spacing between these sections? The file feels a bit cramped. Also am I correct that because PyCTR reads the envrionment variables at start, this can't be done in a `.env`?

Besides that could you just add a note as to what these variables are for?

Reply:
When I added spacing it didn't look any different when viewing the actual markdown ¯\_(ツ)_/¯ but sure, I'll change it

Reply:
> When I added spacing it didn't look any different when viewing the actual markdown ¯_(ツ)_/¯ but sure, I'll change it

That depends on how much spacing you do, but you're right it generally won't. I only said the file is cramped, not the actual rendered HTML. It's not a huge issue, just a request

> It can, but it would require some retooling. I can make it happen if it would be more desirable though

Nah that's alright, it's okay the way it is. Was just making sure I understood is all :+1: